cider/exec/operator/aggregate/CiderAggTargetColExtractorTmpl.h

```diff
@@ -78,14 +78,24 @@ class SimpleAggExtractor : public CiderAggTargetColExtractor {
           CiderBitUtils::clearBitAt(nulls, i);
           ++null_count;
         } else {
-          buffer[i] = *reinterpret_cast<const ST*>(rowPtr + offset_);
+          if (this->getName() == "FLOAT_DOUBLE") {
+            buffer[i] =
+                std::stod(std::to_string(*reinterpret_cast<const ST*>(rowPtr + offset_)));
```

Comment:
Q: why we have to transfer ST to string and then to double? same below

Reply:
from float to double, the direct cast will cause loss of accuracy. For example, 4.6 will be 4.599999904632568, lead to a checker fail

Reply:
I think it's ok to do this as a work around. But actually we should accept such loss of accuracy if we need do float to double conversion. Otherwise we should use decimal.

Reply:
In nextgen, we should won't need to handle these scenarios, which caused by cider internal sum(fp)->fp conflicts with velox sum(fp)->double. We should support support sum(fp)->double directly.